database/seeders/DatabaseSeeder.php

```diff
@@ -33,121 +33,147 @@ public function run(): void
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'manager',
-            'work_space_id' => 2,
-            'email' => $faker->email()
-        ]);
-        User::factory()->create([
-            'role' => 'supplier',
-            'work_space_id' => null
-        ]);
-        User::factory()->create([
-            'role' => 'admin',
-            'work_space_id' => null
-        ]);
         User::factory()->create([
             'role' => 'supplier',
             'work_space_id' => null,
             'email' => 'supplier@stage.com'
         ]);
+        #endregion
 
-        $categories = Category::factory(3)->create();
-
-        $products = Product::factory(50)->create([
+        $products = Product::factory(30)->create([
             'work_space_id' => 1,
-            'price' => $faker->numberBetween(1, 1000),
-            'discount' => $faker->numberBetween(0, 50),
-            'title' => $faker->word(),
         ]);
 
-        $products_second = Product::factory(50)->create([
-            'work_space_id' => 2,
-            'price' => $faker->numberBetween(1, 1000),
-            'discount' => $faker->numberBetween(0, 50),
-            'title' => $faker->word(),
+
+        #region Create categories
+        $categories = Category::factory(3)->create([
+            'work_space_id' => 1
         ]);
+        foreach ($categories as $category) {
+            $subcategories = Category::factory(2)->create([
```

Comment:
Naamgeving in de seeder is consistent, ik zie zowel meerdere woorden aan elkaar zonder hoofdletters, pascal casing en streepjes ertussen, gelieve rechttrekken.